### DISCUSSION TITLE
Passwort anzeigen Login Maske

### DISCUSSION THREAD

**Discussion by moritzvolkmar** [Samedis-care/samedis-care-community#26] [Idea retired / denied]
In der Login-Maske Funktion "Passwort anzeigen" (Augensymbol)

![Bildschirmfoto 2022-10-10 um 10 05 35](https://user-images.githubusercontent.com/80519295/194833823-653e7d06-f2ab-4835-9cff-e1decdff28a6.png)

**Comment by YvesR** [comment]
Das Feature ist mobile-only und/oder abhängig von den Browsereinstellungen.
